spec/controllers/api/dashboards_create_spec.rb

```diff
@@ -0,0 +1,247 @@
+      post :create, params: {
+        "data": {
+          "type": "dashboards",
+          "attributes": {
+            "name": "Cities",
+            "summary": "test dashboard one summary",
+            "description": "Dashboard that uses cities",
+            "content": "test dashboard one description",
+            "published": true,
+            "photo": {
+              "cover": "/photos/cover/missing.png",
+              "thumb": "/photos/thumb/missing.png",
+              "original": "/photos/original/missing.png"
+            },
+            "user-id": "57ac9f9e29309063404573a2",
+            "private": true,
+            "production": true,
+            "preproduction": false,
+            "staging": false,
+            "application": [
+              "fake-app"
+            ]
+          }
+        },
+        loggedUser: USERS[:USER]
+      }
+
+      expect(response.status).to eq(403)
+    end
+
+    it 'with role MANAGER should create the dashboard' do
+      post :create, params: {
+        "data": {
+          "type": "dashboards",
+          "attributes": {
+            "name": "Cities",
+            "summary": "test dashboard one summary",
+            "description": "Dashboard that uses cities",
+            "content": "test dashboard one description",
+            "published": true,
+            "photo": {
+              "cover": "/photos/cover/missing.png",
+              "thumb": "/photos/thumb/missing.png",
+              "original": "/photos/original/missing.png"
+            },
+            "user-id": "57ac9f9e29309063404573a2",
+            "private": true,
+            "production": true,
+            "preproduction": false,
+            "staging": false,
+            "application": [
+              "rw"
+            ]
+          }
+        },
+        loggedUser: USERS[:MANAGER]
+      }
+
+      expect(response.status).to eq(201)
+
+      sampleDashboard = json_response[:data]
+
+      validate_dashboard_structure(sampleDashboard)
+
+      expect(sampleDashboard[:attributes][:application]).to eq(['rw'])
+    end
+
+    it 'without an application value should create the dashboard with the default application value' do
+      post :create, params: {
+        "data": {
+          "type": "dashboards",
+          "attributes": {
+            "name": "Cities",
+            "summary": "test dashboard one summary",
+            "description": "Dashboard that uses cities",
+            "content": "test dashboard one description",
+            "published": true,
+            "photo": {
+              "cover": "/photos/cover/missing.png",
+              "thumb": "/photos/thumb/missing.png",
+              "original": "/photos/original/missing.png"
+            },
+            "user-id": "57ac9f9e29309063404573a2",
+            "private": true,
+            "production": true,
+            "preproduction": false,
+            "staging": false,
+            "application": %w(rw gfw prep)
```

Comment:
Shouldn't this test case provide no applications?

Reply:
I don't see why, the user has these 3 apps, so this should work and be saved as such